shadow-cljs.edn

```diff
@@ -0,0 +1,123 @@
+                [district0x/district-ui-now "1.0.2" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-reagent-render "1.0.1" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-router "1.0.7" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-router-google-analytics "1.0.1" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-smart-contracts "1.0.8" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-web3 "1.3.2" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-web3-account-balances "1.0.2" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-web3-accounts "1.0.7" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-web3-balances "1.0.2" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-web3-sync-now "1.0.3-2" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-web3-tx "1.0.12" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-web3-tx-id "1.0.1" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-web3-tx-log "1.0.13" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/district-ui-window-size "1.0.1" :exclusions [cljsjs/react-with-addons reagent com.cognitect/transit-js]]
+                [district0x/re-frame-ipfs-fx "1.1.1"]
+
+                [refactor-nrepl "2.5.0"]
+                [cider/cider-nrepl "0.24.0"]
+                [cider/piggieback "0.4.1"]]
+
+ :nrepl {:middleware [cider.piggieback/wrap-cljs-repl
+                      refactor-nrepl.middleware/wrap-refactor]}
+
+ :http {:port 3449
+        :host "0.0.0.0"}
+
+
+ :builds {:dev-ui {:target :browser
+                   :modules {:main {:init-fn ethlance.ui.core/init}}
+                   :main ethlance.ui.core
+                   :infer-externs true
+                   :js-options {:js-package-dirs ["./ui/node_modules"]}
+                   :npm-deps false
+                   :output-dir "resources/public/js/"
+                   :optimizations :none
+                   :source-map true
+                   :source-map-timestamp true
+                   :closure-defines {"re_frame.trace.trace_enabled_QMARK_" true
+                                     goog.DEBUG true}}
+          ;; :dev-server {:target :node-script
```

Comment:
server builds are not managed by shadow?

Reply:
yeah it isn't, it will make this PR too big. It already contains district.ui.graphql removal + shadow-cljs for UI.